src/encoded/tests/features/forms.feature

```diff
@@ -6,9 +6,10 @@ Feature: Edit forms
         And I wait for the content to load
         And I click the element with the css selector ".icon-gear"
         And I click the link to "/antibodies/ENCAB728YTO/#!edit"
-        And I wait for an element with the css selector "form.rf-Form" to load
+        And I wait for the content to load
         And I wait for 5 seconds
-        And I fill in "antigen_description" with "It's not a very nice antigen"
+        Then I should see an element with the css selector "form.rf-Form"
+        When I fill in "antigen_description" with "It's not a very nice antigen"
         And I press "Save"
```

Comment:
What bothers me a bit about this is I feel like we’ve had to change selenium tests lately because of timing issues on Saucelabs/Travis, and I’m not sure why. This particular one was because the form took several seconds to appear, and Travis gave up. Yet when I run this test locally, the form comes up right away.

Reply:
Looks like this change is correct to me.